src/app/pages/subjects/new/specific/book-viewer.js

```diff
@@ -7,17 +7,44 @@ import {faCaretDown} from '@fortawesome/free-solid-svg-icons/faCaretDown';
 import useAmazonAssociatesLink from '~/pages/details/common/get-this-title-files//amazon-associates-link';
 import {usePrintCopyDialog} from '~/pages/details/common/get-this-title-files/options';
 import useActiveElementContext, {ActiveElementContextProvider} from '~/contexts/active-element';
+import useLanguageContext from '~/contexts/language';
 import './book-viewer.scss';
 
+const localizedTexts = {
+    'en': {
+        orderPrint: 'Order a print copy',
+        getBook: 'Get the book',
+        viewOnline: 'View online',
+        downloadPdf: 'Download a PDF',
+        instructorResources: 'Instructor resources',
+        studentResources: 'Student resources'
+
+    },
+    'es': {
+        orderPrint: 'Solicite una copia impresa',
+        getBook: 'Consigue el libro',
+        viewOnline: 'Ver en línea',
+        downloadPdf: 'Descargar un PDF',
+        instructorResources: 'Recursos del instructor',
+        studentResources: 'Recursos para estudiantes'
+    }
+};
+
```

Comment:
have we looked into using a proper i18n framework? maybe pickup whatever REX uses?

Reply:
This is a good idea.